airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorTestSuiteOptions.yaml

```diff
@@ -15,8 +15,14 @@ properties:
       - "unitTests"
       - "integrationTests"
       - "acceptanceTests"
+      - "liveTests"
   testSecrets:
     description: "List of secrets required to run the test suite"
     type: array
     items:
       "$ref": "Secret.yaml"
+  connectionOptions:
```

Comment:
nit: should they be testConnections for consistency?

Reply:
Makes sense to me, updated.